src/res_printer.ml

```diff
@@ -3978,6 +3978,38 @@ and printPexpApply ~customLayout expr cmtTbl =
 
 and printJsxExpression ~customLayout lident args cmtTbl =
   let name = printJsxName lident in
+  let hasTailSingleLineComment =
```

Comment:
make sense to me. Will take a look later

Reply:
We better not print `>` in the printJsxProps function, because we have additional logic to determine whether the tag is self-closing, and then choose to print (`>` and `</A>`) or `/>`